execution-engine/Cargo.toml

```diff
@@ -38,6 +38,8 @@ num-traits = { version = "0.2", default-features = false }
 postcard = { version = "0.7.2", features = [ "alloc", "use-std" ] }
 platform-services = { path = "../platform-services" }
 policy-utils = { path = "../policy-utils" }
+psa-crypto = { git= "https://github.com/veracruz-project/rust-psa-crypto.git" }
```

Comment:
This should, of course, be `{ path = "../third-party/rust-psa-crypto/psa-crypto" }`!